src/jsdocs/admins.yaml

```diff
@@ -0,0 +1,15 @@
+paths:
+  # Endpoint to authenticate a user
+  /admin/register:
+    # GET operation
+    post:
+      tags:
+        - admins

```

Comment:
From the other reviews. Something like `adminsRegister.yaml`. Also this doc seems to be missing the body definition. And the tokens should be more explicitly defined this time to indicate the type of token required `adminToken` or `userToken`